catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DuplicateCommands.java

```diff
@@ -99,11 +99,11 @@
     String failedDir;
 
```

Comment:
I can't comment where I want to, but can we get some consistency on these option names.  In particular, it seems typically the style is for the name to be a long name with a --, and the aliases to be short with a - however we have exceptions and inconsistency throughout  our options.

In particular lastmonths and similar not having --, and maybe the -month option should be plural.  Also, temporal, endDate, startDate each have one dash, and the name and alias seem to be reversed.  

I think it also might be nice to make it obvious that an option takes an argument, but that may be inconsistent with other built in karaf command documentation, so I will let that go.